opensearch-jobs.yml

```diff
@@ -182,12 +182,12 @@ instance_groups:
           filters:
           - filtertype: pattern
             kind: regex
-            value: logs*
+            value: logs-app*
           - direction: older
             filtertype: age
             source: creation_date
             unit: days
-            unit_count: 1
```

Comment:
I think we want any index older than 1 day to be read only?

Reply:
i made it two because i'm not 100% sure we won;t get a race condition if we make 2 indexs in a day